setup.py

```diff
@@ -0,0 +1,33 @@
+from distutils.core import setup
+import io
+import os
+
+
+readme = "README.md"
+if os.path.isfile("./README.rst"):
+    readme = "README.rst"
+print("Using %s as readme" % readme)
+
+setup(
+    name='kgitb',
+    packages=['kgitb', 'kgitb.rules'],
+    scripts=['bin/resident'],
+    version='0.0.6',
+    description='A commit message linter',
+    long_description=io.open(readme, encoding="utf-8").read(),
```

Comment:
This will cause resource warnings for Python 3 since you didn't close the file object.